vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -215,6 +215,12 @@ namespace Microsoft.ReactNative
     DOC_DEFAULT("8081")
     UInt16 SourceBundlePort { get; set; };
 
+    DOC_STRING(
+      "When using a @.UseFastRefresh, @.UseLiveReload, or @.UseWebDebugger this controls whether the bundle URL requests inline source maps."
+      "If set, the bundle URL will request the source maps inline (it will improve debugging experience, but for very large bundles it could have a significant performance hit)")
```

Comment:
It's not clear what is meant by "the bundle URL will request", since the url is just data (here and above).
I think what this flag controls is whether the app will request the bundler to provide inline source maps, is that right?
If so can you update the text to clarify? Also, we should point to a link explaining what inline source maps are.